spec/factories/works.rb

```diff
@@ -17,11 +17,12 @@
     state { 'first_draft' }
     collection
 
-    factory :valid_deposited_work do
-      deposited
-      with_keywords
-      with_contributors
-      with_attached_file
+    factory :valid_work do
```

Comment:
Before this we had some specs creating a work factory with keywords, contributors, and an attached file but it wasn't obvious why THOSE three associations were used. It's because they're necessary for validation, so give this a name.